src/geom/ellipse/GetBounds.js

```diff
@@ -15,8 +15,8 @@ var GetBounds = function (ellipse, out)
 {
     if (out === undefined) { out = new Rectangle(); }
 
-    out.x = ellipse.x - ellipse.width;
-    out.y = ellipse.y - ellipse.height;
+    out.x = ellipse.left;
```

Comment:
The issue here is that `ellipse.x - ellipse.width` is different from the desired left bound (`ellipse.left`), since `width` is diameter and not radius.

Alternatively we could do `ellipse.x - ellipse.width / 2`;

Reply:
Never mind, when in Phaser, use Phaser.